### PR TITLE
Orphaned clusters removal periodic job

### PR DESCRIPTION
**Pull request by Tomasz-Smelcerz-SAP**
**Description**

Changes proposed in this pull request:

- Introduces periodic job for removal of orphaned cluster
- Unifies scripts for launching tools (clusters, disks, load balancers)
- Adds tests for periodic jobs

Note for the reviewer: This PR changes jobs configuration to safe mode, dryRun = true.
This is on purpose because the changes can be tested only when they're merged on the master branch.
Once it's merged and a few runs on master confirm that configuration of jobs is correct, we'll switch the configuration to a "production" one with one subsequent PR.

See also #177

### CODE REVIEW

File: prow/jobs/kyma/kyma-integration.yaml [hunk 1]
```diff
@@ -134,9 +135,30 @@ periodics:
...
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1
+      command:
+        - "bash"
+      args:
+        - "-c"
+        - "development/clusters-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=true"
```
Comment (Demonsthere) [inline prow/jobs/kyma/kyma-integration.yaml]: `Clusters-cleanup` or `resource-cleanup` should be executed here?

Comment (Tomasz-Smelcerz-SAP) [reply prow/jobs/kyma/kyma-integration.yaml]: clusters-cleanup.sh
resources-cleanup is a script introduced to reduce repetition, it contains common code.

**Comment by Demonsthere** [comment]
Please add the `resources-cleanup.sh` script to `README.md` and document the parameters its use.

File: development/README.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ The `development` folder has the following structure:
   ├── update-jobs.sh           # This script updates the new configuration of the jobs on a cluster.
   ├── update-plugins.sh        # This script updates the new configuration of the "plugins.yaml" file on a cluster.
   ├── validate-config.sh       # This script runs the "Checker" application.
-  └── validate-scripts.sh      # This script performs a static analysis of bash scripts in the "test-infra" repository.
+  ├── validate-scripts.sh      # This script performs a static analysis of bash scripts in the "test-infra" repository.
+  ├── clusters-cleanup.sh      # This script invokes the tool for cleaning orphaned clusters created by kyma-gke-integration job.
```
Comment (klaudiagrz) [inline development/README.md]:
```suggestion
  ├── clusters-cleanup.sh      # This script invokes the tool for cleaning orphaned clusters created by the "kyma-gke-integration" job.
```

File: development/README.md [hunk 3]
```diff
@@ -20,6 +20,10 @@ The `development` folder has the following structure:
...
+  ├── disks-cleanup.sh         # This script invokes the tool for cleaning orphaned disks created by kyma-gke-integration job.
```
Comment (klaudiagrz) [inline development/README.md]:
```suggestion
  ├── disks-cleanup.sh         # This script invokes the tool for cleaning orphaned disks created by the "kyma-gke-integration" job.
```

File: development/README.md [hunk 4]
```diff
@@ -20,6 +20,10 @@ The `development` folder has the following structure:
...
+  ├── loadbalancer-cleanup.sh  # This script invokes the tool for cleaning orphaned load balancers created by kyma-gke-integration job.
```
Comment (klaudiagrz) [inline development/README.md]:
```suggestion
  ├── loadbalancer-cleanup.sh  # This script invokes the tool for cleaning orphaned load balancers created by the "kyma-gke-integration" job.
```

**Pull request merged**